src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,81 @@
+.questions {
+  display: flex;
+  flex-flow: column;
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(32px 32px 0);
+  }
+
+  &__quit-survey {
+    order: 2;
+    margin-left: auto;
+
+    cursor: pointer;
+  }
+
+  &__container {
+    @include details-container();
+    position: relative;
+
+    flex: 1 1 auto;
+  }
+
+  &__details {
+    margin: auto 0;
+  }
+
+  &__number {
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'lg');
+    line-height: rem(20px);
+
+    opacity: 0.5;
+  }
+
+  &__title {
+    margin: 0;
+    padding-top: rem(10px);
```

Comment:
@rafayet-monon How about change this to `margin-top: rem(16px)`?
![Screen Shot 2563-11-30 at 15 37 19](https://user-images.githubusercontent.com/1772999/100586731-2950d500-3322-11eb-8918-c5a6ad7f4f1e.png)